translations/translations.json

```diff
@@ -5506,7 +5506,7 @@
       "openInApp": "In der {{appName}} App öffnen",
       "open": "Öffnen",
       "view": "Anzeigen",
-      "getOnPlayStore": "Im Google Play Store anschauen",
+      "getOnPlayStore": "Holen Sie es - Im Google Play Store anschauen",
```

Comment:
Oh sorry, that's a misunderstanding. The whole translation value should be "Im Google Play Store anschauen", without that "Holen Sie es". "Holen Sie es" sounds weird. Like a McDonald's ad or something.